internal/search/selector_evaluator.go

```diff
@@ -239,6 +239,9 @@ func (e *SelectorEvaluator) evaluateEq(value any, args []any) (result bool,
 	case float64:
 		arg := arg.(float64)
 		result = value == arg
+	case bool:
+		arg := arg.(bool)
+		result = value == arg
```

Comment:
wonder if there's a cleaner way to handle maybe with another library? feels like we have to keep doing this we find new types (last time you had to tackle float) 🫤